index.html

```diff
@@ -9,225 +9,278 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="flex-container">
-      <header>
-        <a href="" class="white text-normal align-center roboto">Mudasir Sherwani</a>
-        <nav>
+    <header>
+        <a href="" class="welcom-name">Mudasir Sherwani</a>
+          <nav class="flex-container">
+            <ul class="flex-container menu-lists">
+              <li>Portfolio</li>
+              <li>About</li>
+              <li>Contact</li>
```

Comment:
- [ ] [OPTIONAL] Note that these are the navigation links. They are supposed to anchor tags that direct the user to a specific part of the portfolio 💡. Please make these list items links. Note that they should be list items but also links, that means you may just wrap each text into an `<a>`. For example 👇 
```bash
<li><a href='#' >Portfolio</a></li>
//other links
```
Then after you give them the styles so that they look the same as on the Figma design